src/main/resources/com/ncc/aif/restricted_aif.shacl

```diff
@@ -213,6 +213,20 @@ aida:RequiredJustificationPropertyShape
    sh:minCount 1 ;
    sh:message "Type assertions must have at least one justification" .
 
+#########################
+# Justifications require a source document and a source
+# The source restriction is defined on the aida:SharedJustificationShape in the aida_ontology.shacl
+#------------------------
+aida:JustificationSourceDocumentShape
+    a sh:NodeShape ;
+    sh:targetClass aida:Justification ;
+    sh:property [
+        sh:path aida:sourceDocument ;
+        sh:minCount 1 ;
+        sh:message "Justifications require source and a sourceDocument" ;
```

Comment:
We should make this explicit for sourceDocument

Reply:
fixed.